control/freqplot.py

```diff
@@ -60,7 +58,7 @@
 #
 
 # Bode plot
-def bode_plot(syslist, omega=None, dB=None, Hz=None, deg=None,
+def bode_plot(syslist, omega=None, omega_num=None, dB=None, Hz=None, deg=None,
```

Comment:
if backward compatibility is important, `omega_num` should go just before `*args`